content/shared/influxdb3-visualize/superset.md

```diff
@@ -0,0 +1,263 @@
+in {{< product-name >}}.
+
+> Apache Superset is a modern, enterprise-ready business intelligence web application.
+> It is fast, lightweight, intuitive, and loaded with options that make it easy for
+> users of all skill sets to explore and visualize their data, from simple pie
+> charts to highly detailed deck.gl geospatial charts.
+>
+> {{% cite %}}-- [Apache Superset documentation](https://superset.apache.org/docs/intro){{% /cite %}}
+
+> [!Important]
+> **Apache Superset only supports SQL**. You can not use Superset to query
+> {{< product-name >}} with InfluxQL.
+
+<!-- TOC -->
+
+- [Set up Docker for Superset and Flight SQL](#set-up-docker-for-superset-and-flight-sql)
+  - [Install prerequisites](#install-prerequisites)
+  - [Set up Docker for Superset](#set-up-docker-for-superset)
+- [Start the Superset Docker containers](#start-the-superset-docker-containers)
+- [Log in to Superset](#log-in-to-superset)
+- [Create a database connection for InfluxDB](#create-a-database-connection-for-influxdb)
+- [Query InfluxDB with Superset](#query-influxdb-with-superset)
+- [Build visualizations with Superset](#build-visualizations-with-superset)
+
+<!-- /TOC -->
+
+## Set up Docker for Superset and Flight SQL
+
+### Install prerequisites
+
+We recommend using **Docker and docker-compose** to run Superset.
+To set up Superset to run in Docker containers with Flight SQL, follow these steps:
+
+> [!Warning]
+> **Superset** is not officially supported on Windows. For more information about
+> running Superset with Windows and Docker, see the
+> [Superset documentation](https://superset.apache.org/docs/installation/installing-superset-using-docker-compose#1-install-a-docker-engine-and-docker-compose).
+
+1.  Follow the instructions to download and install Docker and docker-compose for your system:
+
+    - **macOS**: [Install Docker for macOS](https://docs.docker.com/desktop/install/mac-install/)
+    - **Linux**: [Install Docker for Linux](https://docs.docker.com/desktop/install/linux-install/)
+
+2.  Follow the [Python.org Downloading Python](https://wiki.python.org/moin/BeginnersGuide/Download)
+    instructions for your system.
+
+3.  Once you've installed Python, check that you can run Python and Pip.
+    Depending on your system, you may need to use the Python version 3 (`python3` and `pip3`) commands.
+    Enter the following commands into your terminal:
+
+    {{< code-tabs-wrapper >}}
+{{% code-tabs %}}
+[python](#)
+[python3](#)
+{{% /code-tabs %}}
+{{% code-tab-content %}}
+```sh
+python --version
+```
+```sh
+pip --version
+```
+{{% /code-tab-content %}}
+{{% code-tab-content %}}
+```sh
+python3 --version
+```
+```sh
+pip3 --version
+```
+{{% /code-tab-content %}}
+    {{< /code-tabs-wrapper >}}
+
+    If neither `pip` nor `pip3` works, follow one of the
+    [Pypa.io Pip Installation](https://pip.pypa.io/en/stable/installation/)
+    methods for your system.
+
+3. Use Pip to install the `flightsql-dbapi` library.
```

Comment:
Oh, I need to update this

Reply:
Do I need to hold this PR for an update?